iocore/hostdb/HostDB.cc

```diff
@@ -2565,18 +2568,9 @@ int HostDBFileUpdateActive = 0;
 // Contents of the host file. We keep this around because other data
 // points in to it (to minimize allocations).
 ats_scoped_str HostFileText;
```

Comment:
This needs to be moved to the host file map, as it needs to have *exactly* the same life time as that map (as all of the `ConstBuffer` items are pointing in to this buffer).